docs_sources/api.md

```diff
@@ -270,7 +270,7 @@ fit(features, classes, sample_weight=None, groups=None)
 <div style="padding-left:5%" width="100%">
 Run the TPOT optimization process on the given training data.
 <br /><br />
-Uses genetic programming to optimize a machine learning pipeline that maximizes the score on the provided features and classes. Performs internal stratified k-fold cross-validaton to avoid overfitting on the provided data.
+Uses genetic programming to optimize a machine learning pipeline that maximizes the score on the provided features and target. This pipeline optimization uses internal k-fold cross-validaton to avoid overfitting on the provided data. The best pipeline is then trained on the entire set of passed samples.
```

Comment:
This pipeline optimization uses --> This pipeline optimization procedure uses

The best pipeline is then trained --> At the end of the pipeline optimization procedure, the best pipeline is then trained

passed samples --> provided